### ISSUE TITLE
Deployed app CORS rules prevent working with web in debug mode

### ISSUE THREAD

**Issue by danieljurek** [Azure/azure-dev#921]
When debugging the web application against a deployed App Service, or Function app we encounter CORS rules configured on the apps themselves that preclude debugging. 

![image](https://user-images.githubusercontent.com/2158838/196763206-2f60735a-cbdb-4ba3-a818-a757c9f3e80f.png)

**Comment by rajeshkamal5050** [comment]
@weikanglim @danieljurek is this still relevant now. Given that we identified debugging from Codespaces is not expected or not supported?

cc @savannahostrowski

**Comment by danieljurek** [comment]
This issue is not codespaces-specific though it will surface in that environment. Removing "codespaces" tag bug keeping this open to ensure proper discussion.

**Comment by rajeshkamal5050** [comment]
@weikanglim is this a must-fix for the upcoming release?

**Comment by weikanglim** [comment]
@rajeshkamal5050 No. I think it's also a rather expensive thing for us to do as part of tooling. I'd prefer to wait until we hear more from users.

**Comment by danieljurek** [comment]
Related to templates and I haven't seen any significant concern on issues like this. This can close.
